tests/Stripe/HttpClient/CurlClientTest.php

```diff
@@ -60,7 +81,7 @@ private function setInitialNetworkRetryDelay($initialNetworkRetryDelay)
 
     private function createFakeRandomGenerator($returnValue = 1.0)
     {
-        $fakeRandomGenerator = $this->createMock('Stripe\Util\RandomGenerator', ['randFloat']);
+        $fakeRandomGenerator = $this->createMock('Stripe\Util\RandomGenerator');
```

Comment:
why this change?

Reply:
The [`createMock()`](https://phpunit.de/manual/5.7/en/test-doubles.html) method only expects a single argument. Don't ask me how it worked before.